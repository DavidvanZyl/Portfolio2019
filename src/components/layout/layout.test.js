import React from "react"
import renderer from "react-test-renderer"

import Layout from "./layout"

describe("Layout", () => {
  test("renders correctly", () => {
    const wrapper = shallow(<Layout children={`<div></div>`} />)
    expect(wrapper).toMatchSnapshot()
  })
})
