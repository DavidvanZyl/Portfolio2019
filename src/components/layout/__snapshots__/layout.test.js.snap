// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<StaticQuery
  data={
    Object {
      "data": Object {
        "site": Object {
          "siteMetadata": Object {
            "title": "David van Zyl",
          },
        },
      },
    }
  }
  query="755544856"
  render={[Function]}
/>
`;
